ui/components/app/network-connection-banner/network-connection-banner.tsx

```diff
@@ -29,23 +32,106 @@ type BannerIcon = {
   className?: string;
 };
 
+const PrimaryMessage = ({
+  t,
```

Comment:
Nit: TSDoc entries would be great for these. I wish we required them for all components, I know it's rare that we document props today though sadly.